Docs/developer_guide/script_repository/sequences.md

```diff
@@ -21,13 +21,13 @@ Get all voxels of a 4D volume (3D volume sequence) as a numpy array called `voxe
 ```python
```

Comment:
Description is inconsistent with example.

Reply:
Following up developer meeting:
* Add another examples for 5D
* Ideally vtkTeemNRRD reader could be fixed to support 5 components

Related to:
* https://github.com/PerkLab/SlicerSandbox/pull/19/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5